components/_host-profiles.scss

```diff
@@ -33,6 +44,36 @@ host profiles.
 	}
 }
 
+.social-container {
+	text-align: center;
```

Comment:
Line should be indented with spaces, not tabs<br>Properties should be ordered padding-top, text-align